functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,18 @@
  */
 struct BeerOrganizer
 {
+    BeerOrganizer(): currentBear{BeerBrand::None} {}
```

Comment:
use in-place initialization 
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c45-dont-define-a-default-constructor-that-only-initializes-data-members-use-in-class-member-initializers-instead

Reply:
Done